THIS REPO IS OUDATED
====================
**Please use the new repo [vedur](https://github.com/axeldavid/vedur) instead.**

I changed the name of this repo from "vedur.is" to "vedur".
I did this because a dot in a python library name can cause conflicts since dots are used in python paths.
